README.md

```diff
@@ -1,383 +1,29 @@
 # Google APIs Client Library for Java
 
-- [Library Maintenance](#maintenance)
```

Comment:
Is this getting moved into other modules?

Reply:
This content all moved into the wiki. I migrated all the content from developers.google.com and it was too much for the README.